packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -321,3 +326,116 @@ def get_app_metrics(dcos_api_session, node: str, container_id: str):
     assert 'datapoints' in app_metrics, 'got {}'.format(app_metrics)
     assert 'dimensions' in app_metrics, 'got {}'.format(app_metrics)
     return app_metrics
+
+
+@pytest.mark.skipif(
+    test_helpers.expanded_config.get('security') == 'strict',
+    reason='Strict mode requires that this test be run with a superuser API session.')
```

Comment:
Why is that? Is that because we did not yet build a `rid` / `action` pair for this?

Reply:
Ah sorry this comment is stale; I'll update it to reflect the fact that the current authorizer module only allows resource providers to launch standalone containers (or, more precisely, only allows JWTs containing specific claims to launch standalone containers). Thx!!

Reply:
@jgehrcke I updated this comment, and added a reference to the relevant JIRA. LMK what you think!